Lib/idlelib/editor.py

```diff
@@ -1052,12 +1054,19 @@ def saved_change_hook(self):
         else:
             title = "untitled"
         icon = short or long or title
-        if not self.get_saved():
+        if not self.get_saved() and not macosx.isCocoaTk():
```

Comment:
Please leave *s that people like me will understand.  See end of new comment.
```suggestion
        if not self.get_saved():
```